packages/flutter_tools/lib/src/build_system/targets/common.dart

```diff
@@ -223,8 +223,6 @@ class KernelSnapshot extends Target {
     bool forceLinkPlatform;
     switch (targetPlatform) {
       case TargetPlatform.darwin_x64:
-      case TargetPlatform.windows_x64:
```

Comment:
This isn't necessary now since we're correctly including the isolate/vm snapshot data. Saves a bit of compile/startup time